R/features.R

```diff
@@ -0,0 +1,65 @@
+features_config <- new.env(parent = emptyenv())
```

Comment:
Would this be cleaner as something like this? I don't have a strong opinion though.
```suggestion
features_config <- rlang::env(
    known_features = tibble::tribble(
        ~Name, ~RequiresPackage, ~Package,
        "ndarray", TRUE, "ndarray",
        "serde", TRUE, "serde",
        "num-complex", TRUE, "num-complex",
        "graphics", FALSE, NA
    )
)
```
```